sysutils/nut/src/opnsense/mvc/app/models/OPNsense/Nut/Nut.xml

```diff
@@ -98,6 +98,10 @@
         <default></default>
         <Required>N</Required>
       </address>
+      <port type="TextField">
+        <default>3493</default>
+        <Required>N</Required>
```

Comment:
Required N doesn't work here if you write the literal ":" into the config unconditionally.

Reply:
Yes, I had not tried to remove the variable.

I have changed the template so that it only adds ":" if the port is defined.

Reply:
best use a PortField type to validate input data

Reply:
Thanks, I have already applied the change.